packages/playground-examples/copy/pt/3-8/Breaking Changes/Checking Unions with Index Signatures.ts

```diff
@@ -0,0 +1,31 @@
+//// { compiler: { ts: "3.8.3" } }
+// Nas versões anteriores do TypeScript, o verificador não valida
+// se os campos não declarados em uma união de tipos estavam em 
+// conformidade com qualquer tipo indexado na união.
+
+// Você pode aprender mais sobre tipagens indexadas aqui: exemplo:indexed-types
```

Comment:
```suggestion
// Você pode aprender mais sobre tipagens indexadas aqui: example:indexed-types
```
`example:*` are hyperlinks and should not be translated